Rock.Messaging.RabbitMQ.IntegrationTests/ConfigTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading;
+using NUnit.Framework;
+
+namespace Rock.Messaging.RabbitMQ.IntegrationTests
+{
+    public class ConfigTests
```

Comment:
Looking at this, I appear to dump to the console on receiving messages- I added an int "numberOfReceivedMessages" and increment that as the consumers receive their messages. 

I also added Assert (sentNumber == receivedNumber) at the end of this test.